vioscsi/vioscsi.c

```diff
@@ -1011,7 +1011,9 @@ VioScsiInterrupt(
         return FALSE;
     }
 
-    RhelDbgPrint(TRACE_LEVEL_VERBOSE, " IRQL (%d)\n", KeGetCurrentIrql());
+    //NOTE : SDV banned function
+    //RhelDbgPrint(TRACE_LEVEL_VERBOSE, " IRQL (%d)\n", KeGetCurrentIrql());
```

Comment:
Why not remove the comment as well? 
Are you planning to use this printout during debugging? 

In any case, @vrozenfe, you can merge if it is good enough for you.

Reply:
@YanVugenfirer, I thought of just removing it, but then thought it could indeed have some utility for debugging.

Although this one's not mine - and I didn't check when it made it's way in (I think it's been there for some time) - I was minded to present it similarly to what I use elsewhere, e.g. [in my rendition of VioScsiFindAdapter()](https://github.com/benyamin-codez/kvm-guest-drivers-windows/blob/bc85387690c04baaf21adad52b2ee827c3f740f3/vioscsi/vioscsi.c#L1373-L1375), i.e.:
```c
//FIXME : SDV banned functions
RhelDbgPrint(TRACE_LEVEL_INFORMATION, " Higest NUMA Node Number : %lu \n", KeQueryHighestNodeNumber());
RhelDbgPrint(TRACE_LEVEL_INFORMATION, " Active CPU KAFFINITY Mask : %I64d \n", KeQueryActiveProcessors());
```
That being said, I'm unlikely to ever raise a PR with the above...
Happy to remove it if either yourself, and / or @vrozenfe, would prefer it was done away with.